packages/components/psammead-sitewide-links/src/List/index.jsx

```diff
@@ -73,16 +73,16 @@ const StyledList = styled.ul`
     padding: ${GEL_SPACING} 0;
     margin-bottom: ${GEL_SPACING};
     grid-column: 1/-1;
-    @supports not (${grid}) {
-      width: 100%;
-    }
+    width: 100%;
   }
 `;
 
 const StyledListItem = styled.li`
+  display: inline-block;
   min-width: 50%;
-  @supports not (${grid}) {
-    display: inline-block;
+
+  @media all and (-ms-high-contrast: none), (-ms-high-contrast: active) {
```

Comment:
Is this to detect we are on IE? What about using `column-gap`, as we did in the `UsefulLinks`?
https://github.com/bbc/psammead/pull/2313/files#diff-72dd5167119d67cc68065f80ca0f141fR39

Reply:
Instead of it, sorry.

Reply:
but then how do we make sure this is only added to IE? that was my issue with this change. The media query looks horrible but it was the only way (that I know of) to make sure the padding is only added to IE

Reply:
Moved this discussion to a private chat and we used a solution similar to the Useful Links IE fix [here](https://github.com/bbc/psammead/blob/latest/packages/components/psammead-useful-links/src/index.jsx#L38-L39).